### PR TITLE
Add loop duration

### PR DESCRIPTION
**Pull request by brozikcz**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -131,14 +137,22 @@ bool handleStatusServerRequests() {
...
     client.flush();
     client.stop();
   }
   return true;
 }
 
+void loop_duration() {
+  unsigned long now = micros();
+  unsigned long delta = now - lastTime;
+  lastTime = now;
```
Comment (danielkucera) [inline src/main.cpp:152]:
```suggestion
  static unsigned long lastTime = 0;
  
  lastTime = now;
```

File: src/main.cpp [hunk 2]
```diff
@@ -22,6 +24,10 @@ WiFiClient enhClients[MAX_SRV_CLIENTS];
 unsigned long last_comms;
 int last_reset_code = -1;
 
+unsigned long lastTime = 0;
```
Comment (danielkucera) [inline src/main.cpp]:
```suggestion
```

File: src/main.cpp [hunk 3]
```diff
@@ -22,6 +24,10 @@ WiFiClient enhClients[MAX_SRV_CLIENTS];
...
+unsigned long loopDuration = 0;
+unsigned long loopCount = 0;
```
Comment (danielkucera) [inline src/main.cpp]:
```suggestion
```
unused variable

File: src/main.cpp [hunk 4]
```diff
@@ -131,14 +137,22 @@ bool handleStatusServerRequests() {
   if (client.availableForWrite() >= AVAILABLE_THRESHOLD) {
     client.printf("uptime: %ld ms\n", millis());
     client.printf("rssi: %d dBm\n", WiFi.RSSI());
-    client.printf("free_heap: %d\n", ESP.getFreeHeap());
+    client.printf("free_heap: %d b\n", ESP.getFreeHeap());
```
Comment (danielkucera) [inline src/main.cpp]:
```suggestion
    client.printf("free_heap: %d B\n", ESP.getFreeHeap());
```

**Review by danielkucera — COMMENTED** [review]
Please check the review.

**Comment by danielkucera** [comment]
Does it work after the changes?

**Comment by brozikcz** [comment]
I supposed you tested on your device, but probably not, now it's fixed and updated.

**Pull request merged**